.github/workflows/deploy-fly.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         working-directory: frontend
         run: npm install && npm run build
       - name: "Copy assets"
-        run: cp -r frontend/build backend/src/main/resources/static
+        run: cp -r frontend/build/ backend/src/main/resources/static
```

Comment:
Wird nicht beschrieben in der PR changelist

Reply:
merk ich mir fürs nächste mal

Reply:
Ich hätte es gerne dieses mal auch

Reply:
done